config/config.php

```diff
@@ -110,98 +110,98 @@
 	'stable' => [
 		'23' => [
 			'100' => [
-				'latest' => '23.0.0',
-				'internalVersion' => '23.0.0.10',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-23.0.0.zip',
+				'latest' => '23.0.1',
+				'internalVersion' => '23.0.1.2',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-23.0.1.zip',
```

Comment:
if it is a stble version and for ```production``` shouldn't it be in the ```releases``` folder instead of ```prereleases``` ?

Reply:
Yes, I don't remember when we decide to move it.
@blizzz can tell us tomorrow :+1:

Reply:
finals needs to be moved to releases, as is documented

Reply:
Done